### PR TITLE
Updating the readme to correct updated env variables and wordsmithing

### PR DESCRIPTION
**Pull request by dcasavant**

### CODE REVIEW

**Comment by dcasavant** [comment]
Some of the default values for the env vars still need to be updated (either in the readme or in the Dockerfile)

File: README.md [hunk 1]
```diff
@@ -1,47 +1,43 @@
 Pega Docker Image
 ===========
 
-This project is a slim version of pega-web docker image that runs pega 8 platform.
-
-
-# Usability note
-
-Although this code supports running Pega8 in a tomcat based docker container, it does not mean that Pega8 can be used in a docker container clustering environment as is, such as Docker Swarm and Kubernetes.
+This project produces a base Docker image that can be used to create a complete Docker image for running Pega using containers.  The Pega .war file is not included in this build.
```
Comment (taz-mon) [inline README.md]: Docker image for running Pega applications using containers

File: README.md [hunk 2]
```diff
@@ -1,47 +1,43 @@
...
 
 # Using this image
 
-The image itself is not runnable directly because it does not come with the Pega 8
-web applications.  Therefore you must use this image as a base to construct an 
-executable Docker image.
+The image itself is not runnable directly because it does not come with the Pega web application (.war file).  Therefore you must use this image as a base to construct an executable Docker image.
```
Comment (taz-mon) [inline README.md]: You cannot run the image directly because...

File: README.md [hunk 3]
```diff
@@ -1,47 +1,43 @@
...
 
 ## Constructing your image
 
-The simplest way to use this image is to create your own Dockerfile with contents similar to the example below and 
-extract the Pega distribution to the same directory as the Dockerfile.  It is recommended that this is done on a Linux system to retain proper file permissions.  Replace the source paths with the actual paths to the Pega 8 software libraries and specify a valid JDBC driver for your target database.
+The simplest way to use this image is to create your own Dockerfile with contents similar to the example below and extract the Pega distribution to the same directory as the Dockerfile.  It is recommended that this is done on a Linux system to retain proper file permissions.  Replace the source paths with the actual paths to the Pega Infinity software libraries and specify a valid JDBC driver for your target database.
```
Comment (taz-mon) [inline README.md]: It is recommended that this is done on a Linux system ==>It's a best practice to configure the deployment on a Linux system

File: README.md [hunk 4]
```diff
@@ -1,47 +1,43 @@
...
 
 Upon successful completion of the above command, you will have a Docker
-image that is registered in your local registry named pega8-tomcat:latest
-and that you can view using the `docker images` command
+image that is registered in your local registry named pega-tomcat:latest
+and that you can view using the `docker images` command.
+
+## Running the image
+
+When running Pega using containers, an orchestration tool is required.  Pega provides support for deployments on Kubernetes using either Helm charts or direct yaml files.  Source code for the deployment scripts may be found in the [pega-helm-charts](https://github.com/pegasystems/pega-helm-charts) repository and more information about Pega on client managed cloud may be found on the [Cloud Choice](https://community.pega.com/knowledgebase/articles/cloud-choice) community site.
```
Comment (taz-mon) [inline README.md]: When running Pega using containers, an orchestration tool is required.  ==> You must use an orchestration tool to run Pega applications using containers. 

You can find the source code for the deployment scripts in the [pega-helm-charts](https://github.com/pegasystems/pega-helm-charts) repository. For information about deploying Pega Platform on a client-managed cloud, see the [Cloud Choice](https://community.pega.com/knowledgebase/articles/cloud-choice community article.

File: README.md [hunk 5]
```diff
@@ -1,47 +1,43 @@
...
 
-# Customizations
+## Image customizations
 
-**JDK and Tomcat**
+**Base image**
 
-Currently the DockerFile extends the base image `tomcat:9-jre11` . This has been tested internally and works fine.
-You can change this to use your preferred Tomcat base image. However this should be thoroughly tested and verfied at your end.There may be some compatibility issue on changing the base image.
+This Docker image extends the base image `tomcat:9-jre11`. This has been extensively validated. You may choose change this to use your preferred Tomcat base image, however any change should be thoroughly tested and verfied.
```
Comment (taz-mon) [inline README.md]: extensively validated ==> thoroughly validated?? looks odd next to "extends the base image ;-)

File: README.md [hunk 6]
```diff
@@ -52,78 +48,55 @@ $ docker run -e "DB_HOST=55.55.55.1" -e "DB_PORT=1234" <image name>[:tags]
...
+**Database connection**
 
 |  Name                        | Purpose                          | Default        |
 | ---------------------------- | -------------------------------- | -------------- |
-| DB_USERNAME                  |                                  | postgres       |
-| DB_PASSWORD | | postgres |
-| DB_HOST | | localhost |
-| DB_PORT | | 5432 |
-| DB_NAME | | postgres |
+| JDBC_DRIVER_URI              | Download (curl) the specified database driver.  If no driver is specified, you must bake a driver into your image.  See *Constructing Your Image* for more information on baking a driver in. | https://jdbc.postgresql.org/download/postgresql-42.1.1.jre7.jar |
```
Comment (taz-mon) [inline README.md]: If no driver is specified, you must bake a driver into your image.  See *Constructing Your Image* for more information on baking a driver in.==>
If you do not specify a driver, you must embed the driver into your Docker image (is it a docker image we're talking about or the JDBC URI???)

**Pull request merged**